frontend/dockerfile/dockerfile_test.go

```diff
@@ -176,6 +176,7 @@ var allTests = integration.TestFuncs(
 	testDockerIgnoreMissingProvenance,
 	testSBOMScannerArgs,
 	testMultiPlatformWarnings,
+	testLintWarnings,
```

Comment:
I think we should have test for each rule and internally cases that cause warnings and cases that don't. Otherwise this will grow very large.

The shared part could still be the same and not duplicated.

Eg.

```
TestLintConsistentCasing() {
cases := []lintTestCase{{ dockerfile: , expectedWarnings: []})
testWarnings(cases)
// testPrintLint(cases)
}
```